musicname/zs_zomboidhouse_v3.cfg

```
@@ -0,0 +1,9 @@
+"music"
+{
+    "[don't fear] the reaper.mp3"                                           "Blue Öyster Cult - (Don’t Fear) The Reaper"
```

Nitpick but use regular apostrophe
```suggestion
    "[don't fear] the reaper.mp3"                                           "Blue Öyster Cult - (Don't Fear) The Reaper"
```